src/components/Main.js

```diff
@@ -120,6 +120,10 @@ class Main extends React.Component {
     }
   }
 
+  showDefaltMainMenuActions() {
```

Comment:
OK, len ked teraz vidis ze je to metoda, tak nevies hned povedat ze ju byuziva iba render. proste je o jeden scope vyssie.